### PR TITLE
Reduce ansible logs when deploy

### PR DESCRIPTION
**Pull request by Starmys**
Do not display "ok" and "skipped" tasks except for verbose mode.

### CODE REVIEW

**Comment by Starmys** [comment]
_TODO:_  do not display profiles of "ok" and "skipped" tasks when `callback_whitelist = profile_tasks`

**Comment by suiguoxin** [comment]
Please introduce `-v` option in the installation doc

**Comment by suiguoxin** [comment]
> _TODO:_ do not display profiles of "ok" and "skipped" tasks when `callback_whitelist = profile_tasks`

Is this TODO item done ?

**Comment by Starmys** [comment]
> > _TODO:_ do not display profiles of "ok" and "skipped" tasks when `callback_whitelist = profile_tasks`
> 
> Is this TODO item done ?

Not yet. I'm looking for how to do it.

File: contrib/kubespray/quick-start-kubespray.sh [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG"
+      exit 1
+      ;;
+  esac
+else
+    export ANSIBLE_DISPLAY_OK_HOSTS=no
+    export DISPLAY_SKIPPED_HOSTS=no
+    export ANSIBLE_CALLBACK_WHITELIST=""
```
Comment (suiguoxin) [inline contrib/kubespray/quick-start-kubespray.sh:17]: what should `ANSIBLE_CALLBACK_WHITELIST` be in `-v` mode ?

Comment (Starmys) [reply contrib/kubespray/quick-start-kubespray.sh:17]: We didn't specified it. In `-v` mode, it can be customized by the user.

**Review by suiguoxin — APPROVED** [review]

**Pull request merged**